### ISSUE TITLE
Look for binary for this platform - undefined

### ISSUE THREAD

**Issue by Goldfinch** [laverdet/node-fibers#393]
Having an issue during webpack compilation, one of the package I use ([camanjs](https://github.com/meltingice/CamanJS)) has fibers in its dependencies.

**OS:** macOS Mojave 10.14
**Node version:** v10.9.0

The issue I'm running into related to the missing binary

```
## There is an issue with `node-fibers` ##
`/bin/undefined-undefined-undefined/fibers.node` is missing.

Try running this to fix the issue: undefined //build
```
According to the error message **process.platform**, **process.arch**, **process.versions.modules** are returning undefined in fibers.js and binary cannot be found. However, when I reinstall fibers I can see testing of an existing binary

```
`darwin-x64-64` exists; testing
Binary is fine; exiting
```

Have I missed something? Would much appreciate any assistance on this.

Thanks

**Comment by Pavel910** [comment]
Have the same issue, I am also using `camanjs`. 
Temporarily bridging the error by loading `camanjs` from CDN, but would love to be able to properly import it in my code.

**Comment by laverdet** [comment]
I've never seen this issue before. I think there is something strange going on with your nodejs installation. How did you install nodejs? Are you trying to use Webpack?

**Comment by Pavel910** [comment]
@laverdet yes, I am using webpack. I'm running macOS Sierra, node `v8.10.0`, sometimes I switch between the node versions using the [n tool](https://www.npmjs.com/package/n). Never had any problems with other binaries :(

**Comment by Goldfinch** [comment]
@laverdet I have reinstalled OS not long time ago and didn't have any conflicts with other packages. The node has been installed through brew, having one of the latest version. I'm using webpack, but when attempting to import/require the package it leads to this issue of node-fibers.

**Comment by laverdet** [comment]
Fibers is a native module, you can't use it with Webpack.

**Comment by Goldfinch** [comment]
@laverdet Yes, I'm aware of it. Sorry, I probably wasn't clear enough by saying _import/require the package_, I meant import/require of `camanjs`. But the thing is webpack, during compilation, is refered to node-fibers problem.

**Comment by Pavel910** [comment]
@laverdet We are trying to use camanjs in the browser :) But apparently it has a dependency on node-fibers when using it from node on the server - so what you say makes sense. You can't just use it as-is in the browser. That is why I switched to the CDN version of camanjs which is built for browser environments.

Thanks for your time!
